docs/DART_LAB/matlab/private/update_inflate.m

```diff
@@ -1,10 +1,16 @@
 function [new_cov_inflate, new_cov_inflate_sd] = update_inflate(x_p, r_var, y_o, sigma_o_2, ss_inflate_base, lambda_mean, lambda_sd, ...
-                                                    inf_lower_bound, inf_upper_bound, gamma_corr, sd_lower_bound_in)
+                                                    inf_lower_bound, inf_upper_bound, gamma_corr, sd_lower_bound_in, ens_size, flavor)
 % Adaptive scheme to update the inflation both in space and time. 
-% Based on the algorithm in the following article:
+% Based on the algorithm in the following articles:
+%
+% [flavor: 2, Gaussian inflation pdf] 
 % Anderson, J. L., 2009: Spatially and temporally varying adaptive covariance 
 % inflation for ensemble filters. Tellus A, 61, 72-83. doi: 10.1111/j.1600-0870.2008.00361.x 
 %
+% [flavor: 5, I-Gamma inflation pdf] 
+% El Gharamti, M., 2018: Enhanced Adaptive Inflation Algorithm for Ensemble Filters. 
+% Monthly Weather Review, 146, 623-640. doi: 10.1175/MWR-D-17-0187.1
+%
```

Comment:
I heavily refactored this on my candidate branch.  Not sure how much to review here ...